packages/builders/package.json

```diff
@@ -53,7 +53,7 @@
 	"homepage": "https://discord.js.org",
 	"dependencies": {
 		"@sapphire/shapeshift": "^3.2.0",
-		"discord-api-types": "^0.33.5",
+		"discord-api-types": "^0.36.0",
```

Comment:
If we bump it here without bumping it everywhere we'll have version clashes again :pepehands